prov/gni/src/gnix_fabric.c

```diff
@@ -53,12 +53,12 @@
 #include "prov.h"
 
 #include "gnix.h"
+#include "gnix_cm_nic.h"
 #include "gnix_util.h"
 #include "gnix_nameserver.h"
 
 const char gnix_fab_name[] = "gni";
 const char gnix_dom_name[] = "/sys/class/gni/kgni0";
-atomic_t gnix_id_counter;
```

Comment:
no not really.  It would be nice to be able to initialize gnix_id_counter statically, i.e. not have to use
```c
atomic_init(&gnix_id_counter,0);
```